website/.github/ISSUE_TEMPLATE/user-feedback.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
What will be the use case of this? I can see this potentially overlapping with the feature request or bug report